client/src/components/Details/Details.stories.js

```diff
@@ -0,0 +1,59 @@
+import { useArgs } from "@storybook/client-api";
+import React from "react";
+
+import { StatelessDetails } from "./Details";
+
+import { Timer } from "./Timer";
+
+export default {
+  title: "Dropdown/Details",
+  component: StatelessDetails,
+
+  // Restricting the control of booleans
+  argTypes: {
+    persist_content: {
+      control: "",
+    },
+    is_open: {
+      control: "",
+    },
+  },
+
+  // Need decorators to use useArgs()
+  decorators: [(Story) => <div>{Story()}</div>],
+};
+
+const Template = (args) => {
+  const [_, updateArgs] = useArgs();
+
+  const on_click = () => {
+    updateArgs({ ...args, is_open: !args.is_open });
+  };
+
+  return (
+    <>
+      <StatelessDetails {...args} on_click={on_click} />
+      <Timer persist_content={args.persist_content} is_open={args.is_open} />
```

Comment:
Oh, haha, when I suggested this I meant putting the time/counter inside the details children. Wouldn't need the props to keep the timer in sync, it's behaviour would be demonstrated because it would reset when closed/opened in one case but continue counting/not reset while closed in the other.

Reply:
I tried to implement it into the content prop, but the timer seems to reset whenever is_open is false. I've put a comment on it for now.